olive/passes/openvino/conversion.py

```diff
@@ -15,37 +15,52 @@
 class OpenVINOConversion(Pass):
     """Converts PyTorch, ONNX or TensorFlow Model to OpenVino Model."""
 
+    _requires_user_script = True
+
     @staticmethod
     def _default_config(accelerator_spec: AcceleratorSpec) -> Dict[str, PassConfigParam]:
         return {
             "input": PassConfigParam(
-                type_=List[Tuple],
+                type_=Union[Callable, str, List],
```

Comment:
not an issue but should we also make this type only as function or function name like we do in other places in Olive. 
The `List` case is only useful when someone want to use the python api but we don't have any use case for that except for the unit tests.

Reply:
This is useful for those simpler model like our current example (even I removed this): https://github.com/microsoft/Olive/blob/main/examples/cifar10_openvino_intel_hw/config.json#L55

Reply:
Oh so, we can still provide the shapes as input?

Reply:
Yes, this is similar as onnx conversion, either input_shape (input) or example_input is needed.

Reply:
My bad, this is still for input shapes. The name `input` was confusing but I guess, that's what the openvino api calls it too.

Reply:
Yeah this is how openvino api calls it. This is so confusing.